dumb-jump.el

```diff
@@ -1705,6 +1705,11 @@ inaccurate jump).  If nil, jump without confirmation but print a warning."
   :group 'dumb-jump
   :type 'boolean)
 
+(defcustom dumb-jump-use-legacy-interface nil
```

Comment:
What do you think of `dumb-jump-disable-obsolete-warnings` instead of `dumb-jump-use-legacy-interface`? To be more specific about what this is actually doing since it's not actually required for using the legacy interface.